src/main/java/io/toast/tk/automation/driver/web/SeleniumHelper.java

```diff
@@ -8,8 +8,8 @@
 import org.openqa.selenium.WebDriver;
 import org.openqa.selenium.WebElement;
 
-import com.synaptix.toast.automation.api.IMiniResult;
-import com.synaptix.toast.core.runtime.IWebElementDescriptor;
+import io.toast.tk.automation.api.IMiniResult;
+import io.toast.tk.core.runtime.IWebElementDescriptor;
 
 public class SeleniumHelper {

```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add a private constructor to hide the implicit public one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.talanlabs.com/coding_rules#rule_key=squid%3AS1118)